src/ca_biositing/pipeline/ca_biositing/pipeline/utils/interactive_commodity_mapper.py

```
@@ -0,0 +1,1313 @@
+
+        if commodity:
+            return {
+                'code': commodity.usda_code,
+                'name': commodity.name,
+                'description': commodity.name,  # Using name as description
+                'source': 'manual_lookup'
+            }
+        else:
+            return None
+
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)